windows/src/engine/keyman32-ver/keyman-debug-etw.h

```diff
@@ -0,0 +1,146 @@
+//**********************************************************************`
```

Comment:
This should not be committed as it is a build artifact; also I think the etw stuff is under one of the proxy dlls now, isn't it?